### PR TITLE
contrib/python-pytest-asyncio: new package (0.23.3)

### PR DESCRIPTION
**Pull request by firefly-cpp**
https://github.com/pytest-dev/pytest-asyncio

### CODE REVIEW

File: contrib/python-pytest-asyncio/template.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+pkgname = "python-pytest-asyncio"
+pkgver = "0.23.4a2"
```
Comment (nekopsykose) [inline contrib/python-pytest-asyncio/template.py:2]: 0.23.3 is the latest stable version

Comment (firefly-cpp) [reply contrib/python-pytest-asyncio/template.py]: The problem is that CI forces to use the latest. How to skip it?

Comment (nekopsykose) [reply contrib/python-pytest-asyncio/template.py]: ignore = ["*a*"] in update.py

File: contrib/python-pytest-asyncio/template.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+license = "Apache-2.0"
+url = "https://github.com/pytest-dev/pytest-asyncio"
+source = f"$(PYPI_SITE)/p/pytest-asyncio/pytest-asyncio-{pkgver}.tar.gz"
+sha256 = "af313ce900a62fbe2b1aed18e37ad757f1ef9940c6b6a88e2954de38d6b1fb9f"
+# missing deps
+options = ["!check"]
+
+
+def post_install(self):
+    self.install_license("LICENSE")
```
Comment (nekopsykose) [inline contrib/python-pytest-asyncio/template.py:23]: don't need to install apache

**Comment by nekopsykose** [comment]
the commit message also still says `0.23.4a2`

**Comment by firefly-cpp** [comment]
It is OK now

**Comment by nekopsykose** [comment]
that changed the title and not the commit

**Comment by firefly-cpp** [comment]
fixed

**Pull request merged**